packages/client/src/components/measurements/MeasurementStatisticsBelowChart.re

```diff
@@ -0,0 +1,131 @@
+[@react.component]  
+let make =  
+    (
+        ~cdf: ForetoldComponents.Types.Dist.t, 
+        ~measurable: Types.measurable
+    ) => {
+    
+    module Calculate = {
+        let toPdf = (cdf: ForetoldComponents.Types.Dist.t) => {
+            let ys =  cdf.ys;
+            let ysPdf = Array.mapi((index, element) => {
+            switch(index){
+                | 0 => element;
+                | someOtherIdex => element -. Array.get(ys,index-1);
+            }
+
+            }, ys); 
+            
+            ysPdf;
+        } 
+        let expectedValue = (cdf: ForetoldComponents.Types.Dist.t) => {
+            let ys= toPdf(cdf);
+            let result = Js.Array.reducei((accumulator, x, index) => (accumulator +. (x *. Array.get(ys,index)) ), 0.0, cdf.xs)
```

Comment:
I think this makes the assumption that samples are distributed across the x-axis evenly, which may not always be the case.